tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/BrowserBased/SolutionUpdateBrowserMobileFirstTests.cs

```diff
@@ -103,7 +104,8 @@ public void ShouldThrowWhenSolutionNotPresent()
         [Test]
         public void CommandShouldTrimStrings()
         {
-            var command = new UpdateSolutionBrowserMobileFirstCommand("Sln1", "     yes     ");
+            var originalViewModel = new UpdateBrowserBasedMobileFirstViewModel { MobileFirstDesign = "     yes     "};
+            var command = new UpdateSolutionBrowserMobileFirstCommand("Sln1", originalViewModel.MobileFirstDesign);
```

Comment:
If a command requires an interface (though it looks like this one does not use an interface) we could look at mocking it rather than using the view model implementation from the API.